docs/content/en/docs/reference/packagespec/harbor/v2.5.1.md

```diff
@@ -0,0 +1,59 @@
+| `expose.loadBalancer.sourceRanges`                                   | List of IP address ranges to assign to loadBalancerSourceRanges                                                                                                                                                                                                                                                                                                                                                                                                   | []                                    |
+| **Internal TLS**                                                     |                                                                                                                                                                                                                                                                                                                                                                                                                                                                   |                                       |
+| `internalTLS.enabled`                                                | Enable TLS for the components (core, jobservice, portal, registry)                                                                                                                                                                                                                                                                                                                                                                            | `true`                               |
+| **Persistence**                                                      |                                                                                                                                                                                                                                                                                                                                                                                                                                                                   |                                       |
+| `persistence.resourcePolicy`                                         | Setting it to `keep` to avoid removing PVCs during a helm delete operation. Leaving it empty will delete PVCs after the chart deleted. Does not affect PVCs created for internal database and redis components.                                                                                                                                                                                                                                                   | `keep`                                |
```

Comment:
```suggestion
| `persistence.resourcePolicy`                                         | Setting it to `keep` to avoid removing PVCs during a helm delete operation. Leaving it empty will delete PVCs after the chart is deleted. Does not affect PVCs created for internal database and redis components.                                                                                                                                                                                                                                                   | `keep`                                |
```